composer.json

```diff
@@ -28,10 +28,11 @@
 	},
 	"require": {
 		"php": ">=7.1",
-		"composer/installers": "1.*,>=1.0.1"	
+		"composer/installers": "1.*,>=1.0.1",
+		"easyrdf/easyrdf": "*",
```

Comment:
Are these two things needed? (And below)

Reply:
@thomas-topway-it ^

Reply:
no, I've removed them